Sources/SwiftFormatCore/Context.swift

```diff
@@ -76,7 +76,8 @@ public final class Context {
     self.fileURL = fileURL
     self.importsXCTest = .notDetermined
     let tree = source.map { Parser.parse(source: $0) } ?? sourceFileSyntax
-    self.sourceLocationConverter = SourceLocationConverter(file: fileURL.relativePath, tree: tree)
+    self.sourceLocationConverter =
+      SourceLocationConverter(fileName: fileURL.relativePath, tree: tree)
```

Comment:
Nitpick: should it be formatted in the following way:
```suggestion
    self.sourceLocationConverter = SourceLocationConverter(
        fileName: fileURL.relativePath,
        tree: tree
    )
```

Reply:
While swift-format supports that, I personally prefer breaking before the function call if it means the entire function call can remain on its own line.